packages/react-components/src/atoms/Button.tsx

```diff
@@ -46,6 +46,7 @@ interface LinkStyleButtonProps {
 }
 type ButtonProps = (NormalButtonProps | LinkStyleButtonProps) & {
   readonly submit?: boolean;
+  readonly label?: string;
```

Comment:
unused?

Reply:
Whoops, Yeah. I made it work better with a label.